src/SprykerMiddleware/Zed/Process/Business/Process/Processor.php

```diff
@@ -12,47 +13,57 @@ class Processor implements ProcessorInterface
      */
     protected $iterator;
 
+    /**
+     * @var \SprykerMiddleware\Zed\Process\Business\Aggregator\AggregatorInterface
+     */
+    protected $aggregator;
+
     /**
      * @var \SprykerMiddleware\Zed\Process\Business\Pipeline\PipelineInterface
      */
     protected $pipeline;
 
     /**
-     * @var \SprykerMiddleware\Zed\Process\Business\Process\Hook\PreProcessorHookInterface[]
+     * @var \SprykerMiddleware\Zed\Process\Business\Process\Hook\PreProcessorHookPluginInterface[]
      */
     protected $preProcessStack;
 
     /**
-     * @var \SprykerMiddleware\Zed\Process\Business\Process\Hook\PostProcessorHookInterface[]
+     * @var \SprykerMiddleware\Zed\Process\Business\Process\Hook\PostProcessorHookPluginInterface[]
      */
     protected $postProcessStack;
 
     /**
-     * @var \Psr\Log\LoggerInterface
-     */
-    protected $logger;
-
-    /**
+     * Processor constructor.
```

Comment:
No need in this.

Reply:
Yea, this is a PhpStorm default, he has to remove it from the code templates there.